lib/geocoder/lookups/google.rb

```diff
@@ -27,6 +27,13 @@ def query_url(query)
 
     private # ---------------------------------------------------------------
 
+    def configure_ssl!(client)
+      client.instance_eval {
+        @ssl_context = OpenSSL::SSL::SSLContext.new
+        @ssl_context.set_params({:options=> OpenSSL::SSL::OP_NO_SSLv2 | OpenSSL::SSL::OP_NO_SSLv3 | OpenSSL::SSL::OP_NO_COMPRESSION})
```

Comment:
Why do we need to change the SSL context? Also is there a reason that this is only relevant to google's API?